server/src/testE2E/java/com/flipkart/varadhi/E2EBase.java

```diff
@@ -74,6 +74,11 @@ static List<Project> getProjects(Response response) {
         });
     }
 
+    static List<String> getTopics(Response response) {
```

Comment:
check if its possible to fold above method in single generic one ? Not sure if it was missed earlier.

Reply:
I checked, but could not find a way to implement a generic method which returns a List of generic Type.